common-pg/src/main/scala/com/socrata/pg/analyzer2/CostEstimator.scala

```diff
@@ -21,7 +22,7 @@ object CostEstimator {
         throw new Exception("EXPLAIN should have returned a single result?");
       }
 
-      val result = JsonReader.fromString(rs.getString(0)).dyn(0).Plan("Total Cost").! match {
+      val result = JsonReader.fromString(rs.getString(1)).dyn(0).Plan("Total Cost").! match {
```

Comment:
...why on earth did _this_ change when the sql didn't?

Reply:
Because it was wrong before.  Result indices start from 1, not 0.